src/components/main/files/browser/styles.scss

```diff
@@ -16,3 +17,69 @@
     }
   }
 }
+
```

Comment:
Since styles are globally scoped, please move everything into the #files scope if you'd like I can send you an example for how it should be best done